src/widgets/histogram/content-view.js

```diff
@@ -134,6 +138,7 @@ module.exports = cdb.core.View.extend({
 
     if (isDataEmpty) {
       this._addPlaceholder();
+      this._initTitleView();
```

Comment:
I suppose initially was on different classes to divede&conquer, but really the title view is really doing a few things. I will change.